docs/authentication/destination-service-authentication.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: mTLS Authentication using the Destination Service
+parent: Authentication
+nav_order: 2
+permalink: /authentication/mtls-authentication-using-the-Destination-service
+---
+
+# mTLS Authentication using the Destination Service
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+   {:toc}
+
+---
+
+## Description
+
+DestinationConfiguration is a class in the context of Alert Notification service node client. It is used to trigger the mTLS authentication using the Destination service, you just need to provide the clientID, client secret, oAuth url, destination service url generated from your Destination Service instance (e.g. from a Service Key or Service Binding) and the name of the destination. For further information go to [Credential Management](https://help.sap.com/docs/ALERT_NOTIFICATION/5967a369d4b74f7a9c2b91f5df8e6ab6/80fe24f86bde4e3aac2903ac05511835.html?locale=en-US) and select Mutual TLS Authentication with Destination Service option.
+
+## Constructor properties
+
+[comment]: <> For loop must remain on the lines, if changed table won't behave normally
+
+| Field | Type | Description |
+|:---:|:--:|:---------:| {% for field in site.data.certificate-service-authentication.propertyFields %}
+| {{field.name}} | {{ field.type }} | {{field.description}} | {% endfor %}
+
+## Methods
+
+[comment]: <> For loop must remain on the lines, if changed table won't behave normally
+
+| Name | Returns | Description |
+|:---:|:--:|:---------:| {% for method in site.data.certificate-service-authentication.methods %}
+| {{method.name}} | {{method.returnValue}}|{{method.description}} | {% endfor %}
+
+## @Example
+
+```js
+import { DestinationConfiguration } from '@sap_oss/alert-notification-client';
+
+const destinationConfiguration = new DestinationConfiguration({
+    username: 'test-certificate',
+    password: 'test-privateKey', 
+    oAuthTokenUrl: 'https://test.oauth.cert.service.com/oauth/token',
+    destinationUrl: 'https://destination-configuration.test.com',
+    destinationName: 'test-destination'
+});
+
+destinationConfiguration.getAuthentication()
+        .then(authentication => console.log(authentication))
+        .catch(error => console.log(error));  // The current call will print the basic authorization header value with encoded username and password in base64 format 'Basic dGVzdC11c2VybmFtZTp0ZXN0LXBhc3N3b3Jk'
```

Comment:
The current call will print the basic authorization header value with Base64-encoded username and password, e.g. 'Basic dGVzdC11c2VybmFtZTp0ZXN0LXBhc3N3b3Jk'